invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposit/custom_fields/CustomAutocompleteDropdown.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+import { FieldLabel, RemoteSelectField } from "react-invenio-forms";
+import _get from 'lodash/get';
+import _isArray from 'lodash/isArray';
+import { Field } from 'formik';
+
+export default class CustomAutocompleteDropdown extends Component {
```

Comment:
wondering if somehow this can be reusable for othervocabularies (not just custom fields ones). e.g. subjects, names